src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerDescriptionProfile.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
+{
+    public class RpcWorkerDescriptionProfile
```

Comment:
nit: move this to `Microsoft.Azure.WebJobs.Script.Workers` and rename to `WorkerDescriptionProfile`